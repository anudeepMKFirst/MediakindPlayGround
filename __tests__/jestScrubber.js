import React from 'react';
import Scrubber from "./../scrubber.youi.js";
import renderer from 'react-test-renderer';

it('renders correctly', () => {
  const tree = renderer
    .create(<Scrubber/>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});