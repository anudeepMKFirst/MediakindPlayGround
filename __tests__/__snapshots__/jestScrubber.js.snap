// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Array [
      Object {
        "flexDirection": "row",
        "height": 8,
        "top": -241,
        "width": 920,
      },
      Object {
        "opacity": 0,
      },
    ]
  }
>
  <View
    style={
      Array [
        Object {
          "backgroundColor": "#D5A23E",
        },
        Object {
          "flex": NaN,
        },
      ]
    }
  />
  <Image
    source={
      Object {
        "uri": "res://drawable/default/Scrubber-Thumb.png",
      }
    }
    style={
      Array [
        Object {
          "height": 24,
          "top": -8,
          "width": 24,
        },
        Object {
          "opacity": 1,
        },
      ]
    }
  />
</View>
`;
